scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
@KjellKod just fyi, for this current branch:

git rev-parse --abbrev-ref HEAD

yields update/add_netmon_build_scripts, which is why the build didn't work when I used it. This applies to any other special chars that shouldn't be in path/file names. I'm sure it can be worked around somehow, it was just easier to hard code it for now.